config/i18n-tasks.yml

```diff
@@ -15,19 +15,23 @@ data:
 
   # Locale files or `File.find` patterns where translations are read from:
   read:
+    - 'config/locales/devise.%{locale}.yml'
+    - 'config/locales/%{locale}.yml'
     ## Default:
     # - config/locales/%{locale}.yml
     ## More files:
     # - config/locales/**/*.%{locale}.yml
 
   # Locale files to write new keys to, based on a list of key pattern => file rules. Matched from top to bottom:
   # `i18n-tasks normalize -p` will force move the keys according to these rules
-  write:
+  # write:
     ## For example, write devise and simple form keys to their respective files:
     # - ['{devise, simple_form}.*', 'config/locales/\1.%{locale}.yml']
     ## Catch-all default:
     # - config/locales/%{locale}.yml
```

Comment:
Let's get rid of these comments if we are using the `write:` key below.

Reply:
I left is as an example, for later reference - these comments are the defaults.

Reply:
We can always regenerate the file if need be. Let's remove these lines.

Reply:
ok - np I'll remove it :)